src/OpenTelemetry.Api/Trace/ActivityExtensions.cs

```diff
@@ -97,6 +97,7 @@ public static Status GetStatus(this Activity? activity)
     /// <remarks> The exception is recorded as per <a href="https://github.com/open-telemetry/opentelemetry-specification/blob/main/specification/trace/exceptions.md">specification</a>.
     /// "exception.stacktrace" is represented using the value of <a href="https://learn.microsoft.com/dotnet/api/system.exception.tostring">Exception.ToString</a>.
     /// </remarks>
+    [Obsolete("Call Activity.AddException instead this method will be removed in a future version.")]
```

Comment:
can we explicitly call out that the end result is same when using the suggested alternative?